### ISSUE TITLE
stdin stderr support

### ISSUE THREAD

**Issue by ozanh** [d5/tengo#273]
AFAIK, current tengo stdlib does not support reading/writing using stdin or stderr. Adding `stdin()` and `stderr()` functions, which return a file like object, to `os` library will be suffice to handle basic requirements. After implementing shebang support these functions will be required by users to use tengo as a standalone scripting language.

**Comment by d5** [comment]
We should also add `stdout()` if we do this. One thing to note is that sandboxing is important in Tengo. In many cases, direct access to stdin/out/err will be less desirable if it's running an arbitrary user code. For example, functions in `fmt` package can be easily controlled because you can simply make it un-importable. Not saying that we should put stdin/out/err in a separate package, but we will need something like that for the sandboxed environments.

**Comment by ozanh** [comment]
I agree with you. I was thinking about creating three functions `stdin()` `stdout()` and `stderr()` under `os` library like `os.open()`. What do you think?

**Comment by d5** [comment]
Yeah. `os` package will not be included for most sandboxed environments anyway.

**Comment by ozanh** [comment]
I will implement this if community shows an interest in it. Other tengo improvements need to be done before this.
